modules/installation-osp-kuryr-known-limitations.adoc

```diff
@@ -5,20 +5,26 @@
 [id="installation-osp-kuryr-known-limitations_{context}"]
 = Known limitations of installing with Kuryr
 
-There are known limitations when using Kuryr SDN:
+There are known limitations when using Kuryr SDN, depending on the
+{rh-openstack} version:
 
 * An Amphora load balancer VM is deployed per OpenShift Service with
 the default Octavia load balancer driver (Amphora driver). If the environment is
-resource constrained, creating a large amount of Services could be a problem.
+resource constrained, creating a large amount of Services could be a problem as
+one VM is created for each Service.
 * Depending on the Octavia version, UDP listeners are not supported. This means
-that OpenShift UDP Services are not supported.
-* There is a known limitation of Octavia not supporting listeners on different
-protocols, like UDP and TCP, on the same port. Thus, Services exposing the same
-port for different protocols are not supported.
+that OpenShift UDP Services are not supported. UDP listeners are supported from
+{rh-openstack} 16+.
+* Depending on the Octavia version, there is a known limitation of Octavia
+not supporting listeners on different protocols, like UDP and TCP, on the
+same port. Thus, Services exposing the same port for different protocols are
+not supported. Listeners on different protocols are supported from
+{rh-openstack} 16+.
 * Due to the above UDP limitations of Octavia, Kuryr forces Pods to use TCP
-for DNS resolution. This is set with the `use-vc` option in `resolv.conf`. This
-might be a problem for Pods running Go applications compiled with the `CGO_DEBUG`
-flag disabled, as that uses the `go` resolver that only leverages UDP and is not
-considering the `use-vc` option added by Kuryr to the `resolv.conf`. This is a
-problem also for musl-based containers as its resolver does not support the
-`use-vc` option. This includes images built from `alpine`.
+for DNS resolution if the {rh-openstack} release is not 16 or newer. This is
```

Comment:
"for DNS resolution in {rh-openstack} versions 15 or earlier. Set this option by changing the value of `use-vc" in `resolve.conf`. Because they use the `go` resolver, which relies on UDP, Pods that run Go applications that were compiled with the `CGO_DEBUG` flag disabled may experience problems [which problems?] unless this option is set. musl-based containers, including those based on `alpine`, do not support the `use-vc` option."

I might need to take another pass at this, but does it preserve the meaning?

Reply:
no it does not. I'll adapt a bit